apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/resources/GenerateTransformersFromDslMojoIT/expected/FissClaimTransformer.java

```diff
@@ -114,7 +114,7 @@ public void transformMessageArraysToFissClaim(gov.cms.mpsm.rda.v1.fiss.FissClaim
       final String itemNamePrefix = namePrefix + "procCodes" + "-" + index + "-";
       final FissProcedureCode itemFrom = from.getFissProcCodes(index);
       final FissProcCode itemTo = transformMessageToFissProcCode(itemFrom,transformer,now,itemNamePrefix);
-      itemTo.setDcn(from.getDcn());
+      itemTo.setDcn(to.getDcn());
```

Comment:
This seems like it would work fine, but the source should still have the DCN shouldn't it?

Reply:
This change was necessary because while the `FissClaim` message does have a`dcn` field, the `FissPayer` message does not have a `claimId` field.  The transformer needs to use the parent entity to copy down the value into the child entity rather than the parent message.  The old logic worked OK but this logic is more correct.